src/pages/hyejinKim/Login/Login.js

```diff
@@ -1,7 +1,64 @@
+import React, { useState } from 'react';
+import { useNavigate } from 'react-router-dom';
+import LoginForm from './LoginForm';
 import './Login.scss';
 
+// 질문
+// 1. 현재 state를 가져오는 경우는 언제인지? 콜백함수가 인자로 들어오면??
+// 2. 제어 컴포넌트랑 무슨 차이지?
+
 function Login() {
-  return <></>;
+  const navigate = useNavigate();
+  const [values, setValues] = useState({
+    id: '',
+    pw: '',
+  });
+
+  const handleInputValue = e => {
+    const { name, value } = e.target;
+    setValues(preValues => ({ ...preValues, [name]: value }));
```

Comment:
계산된 속성명 굳
함수형 업데이트로 하고 계신데,
함수형 업데이트와 값만 넣는 setState는 이전의 state 값을 통해 새로운 state로 update 하는지 여부에 따라 구분해 사용
따라서 이 경우에는 이전 state 값을 통해 update(ex. 기존 state에 +1)하는 것이 아니기 때문에
```suggestion
    setValues({ ...values, [name]: value }));
```
위와 같이 사용해도 무방함

Reply:
values라는 state명이 명확하지 않고, e.target.value와 헷갈려 보임
따라서 state명을 명확하게 수정

Reply:
inputValues로 하려다가 좀더 명확하게 해보려고
로그인 페이지에서는 loginValues, 회원가입 페이지에서는 signupValues로 수정했습니다!